### ISSUE TITLE
[Feature Request] Update to 1.18.2

### ISSUE THREAD

**Issue by Serverfrog** [Draconic-Inc/Draconic-Evolution#1646]
As the Titel says, an update to 1.18.2 would be awesome (due to the fact that now some mod packs are starting to go to this version

**Comment by FoxMcloud5655** [comment]
This is not being considered right now and has been asked multiple times already in the Discord server.  The port to 1.16.5 must be complete before even thinking about updating to another version.

**Comment by Serverfrog** [comment]
Wouldn't this ticket then not still be worth keeping open if it will be done but not now?

**Comment by FoxMcloud5655** [comment]
GitHub Issues is for keeping track of bugs and enhancements.  What you're asking for is a port that may not even be made in the future.  By the time Draconic Evolution is ready to port to the newer versions, 1.18.x may not be the ideal version to port to.  Those kinds of mod packs are bleeding edge right now, not stable.  1.16.5 is the stable modding version.

**Comment by Serverfrog** [comment]
I know for what Issues are, and for me this would be something that is an enhancement for the future. So the first part ist kinda wrong.
Not stable is also kinda wrong due to fact that im currently having a bigger Modpack in Use wich is 1.18.2 and i did not noticed one bug there. But do your issue keeping like you want

**Comment by H4ckerxx44** [comment]
Pinning this issue would maybe help to prevent new issues on the 1.18.x topic in the first place.

**Comment by FoxMcloud5655** [comment]
I'm not talking about stable mods, I'm talking about a stable modding version.  The state of mods and how buggy they are is up to the mod author.  Porting is not the same as enhancements.  Porting targets completely different versions and requires much more time and effort; enhancements add onto existing code and are much easier to create.

I can agree with pinning this issue, however.  Unfortunately, I do not have that access, so it's up to @brandon3055 on what to do.

**Comment by Serverfrog** [comment]
The pinning was also my Idea, im not that much on Github and more on Jira so i didn't know this is a feature here.
But in the Software development sense, this is still a Task that can be tracked, will come up often und should be discussed/thrown away if their time is ( for example latest isn't 1.18 or so on). But thanks for trying to be helpfull in the sense that it seemed like i never saw software development
